export_options.go

```diff
@@ -41,16 +43,40 @@ func NewExportOptionsGenerator(xcodeProj *xcodeproj.XcodeProj, scheme *xcscheme.
 	return g
 }
 
-// GenerateExportOptions generates an exportOptions.plist file.
-func (g ExportOptionsGenerator) GenerateExportOptions(exportMethod exportoptions.Method, containerEnvironment string, teamID string, uploadBitcode bool, compileBitcode bool, xcodeManaged bool,
+// GenerateApplicationExportOptions generates exportOptions for an application export.
+func (g ExportOptionsGenerator) GenerateApplicationExportOptions(exportMethod exportoptions.Method, containerEnvironment string, teamID string, uploadBitcode bool, compileBitcode bool, xcodeManaged bool,
 	xcodeMajorVersion int64) (exportoptions.ExportOptions, error) {
-	bundleIDEntitlementsMap, err := g.projectEntitlementsByBundleID(g.xcodeProj, g.scheme, g.configuration)
+	mainTarget, err := archivableApplicationTarget(g.xcodeProj, g.scheme, g.configuration)
```

Comment:
What if an App Clip Scheme is selected, how will that be handled?

Reply:
As we decided to support only application export for this step, I will make the step failing if scheme selected with app clip target and also print a workaround message (export the application with xcode-archive step and use export-xcarchive step to export the app clip IPA).

Reply:
Something like this:

```
Selected scheme: 'Fruta iOS Clip' targets and App Clip target (Fruta iOS Clip),
'Xcode Archive & Export for iOS' step is intended to archive the project using a scheme targeting an Application target.
Please select a scheme targeting an Application target to archive and export the main Application
and use 'Export iOS and tvOS Xcode archive' step to export an App Clip.
```

Reply:
```Selected scheme: 'Fruta iOS Clip' targets an App Clip target (Fruta iOS Clip).```
and -> an

Reply:
fixed